### PR TITLE
DP-2390 Hugo Versioning

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: .github/actions/hugo-build-versioned-action/action.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Hugo Versioned Build
+description: Builds documentation using Hugo
+inputs:
+  base-url:
+    required: false
+    description: Optional base url used during build
+  working-directory:
+    required: false
+    description: Path to documentation directory
+    default: versioned_docs
```
Comment (hkad98) [inline .github/actions/hugo-build-versioned-action/action.yaml:10]: You do not use there this input variable. Why not?

File: .github/actions/hugo-build-versioned-action/action.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      working-directory: docs
+      run: npm install
+      shell: bash
+    - name: Build
+      working-directory: docs
+      env:
+        HUGO_ENV: ${{ inputs.hugo-env }}
+        BASE_URL: ${{ inputs.base-url }}
+      run: hugo --minify ${BASE_URL:+--baseURL $BASE_URL}
+      shell: bash
```
Comment (hkad98) [inline .github/actions/hugo-build-versioned-action/action.yaml:39]: If I am not mistaken, this is only a repetition of our action. Why not to use it again? 

https://github.com/gooddata/gooddata-python-sdk/blob/master/.github/actions/hugo-build-action/action.yaml

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR213

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/45/ : SUCCESS in 1m 19s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/195/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/198/ : SUCCESS in 15s

**Pull request merged**